upload.py

```diff
@@ -9,17 +9,18 @@
 
 import argparse
 import boto3
-import os
+from env import check_get_env
```

Comment:
Getting a 'module not found' error; should this read scripts/env or is there some other setup missing?

Reply:
replace with
```
from scripts.env import check_get_env
```
and it will work